### PR TITLE
imrpove dropdown & tabs docs

### PR DESCRIPTION
**Pull request by baiyaaaaa**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.
#1917  #1241

### CODE REVIEW

File: examples/docs/en-US/tabs.md [hunk 1]
```diff
@@ -125,6 +128,30 @@ Border card tabs.
 
 :::
 
+### Custom Tab
+
+you can use `label-content` property to customize the tab
```
Comment (Leopoldthecoder) [inline examples/docs/en-US/tabs.md]: 首字母大写啊亲

File: examples/docs/en-US/tabs.md [hunk 2]
```diff
@@ -125,6 +128,30 @@ Border card tabs.
...
+
+:::demo `label-content` is a render function，which return the vnode of the tab。
```
Comment (Leopoldthecoder) [inline examples/docs/en-US/tabs.md]: `，` => `, `
`。` => `.`

File: examples/docs/zh-CN/dropdown.md [hunk 3]
```diff
@@ -43,6 +43,9 @@
     methods: {
       handleClick() {
         alert('button click');
+      },
+      handleCommand(command) {
+        this.$message('click on item ' + command);
```
Comment (Leopoldthecoder) [inline examples/docs/zh-CN/dropdown.md]: `handleCommand` 在下面的 `demo` 块里也写一下吧，否则 jsfiddle 就报错了。英文文档同。

File: examples/docs/en-US/dropdown.md [hunk 4]
```diff
@@ -126,6 +129,27 @@ Use `hide-on-click` to define if menu closes on clicking.
 ```
 :::
 
+### Command event
+
+点击菜单项后会触发事件，用户可以通过相应的菜单项 key 进行不同的操作
```
Comment (Leopoldthecoder) [inline examples/docs/en-US/dropdown.md]: Clicking each dropdown item fires an event whose parameter is assigned by each item.

**Comment by baiyaaaaa** [comment]
done

**Pull request merged**